File: QA/viasoft.qualidade.rnc.gateway/Viasoft.Qualidade.RNC.Gateway.Host/Proxies/Producao/OrdensProducao/Dtos/OrdemProducao.cs
namespace Viasoft.Qualidade.RNC.Gateway.Host.Proxies.Producao.OrdensProducao.Dtos;

public class OrdemProducao
{
    public int Odf { get; set; }
    public string Revisao { get; set; }
    public ProdutoDto ProdutoDTO { get; set; }
    public decimal QuantidadeOrdem { get; set; }
    public decimal QuantidadeProduzida { get; set; }
    public decimal QuantidadeAProduzir { get; set; }
    public bool OrdemServico { get; set; }
    public int IdRoteiro { get; set; }
    public int CodigoRoteiro { get; set; }
    public string DescricaoRoteiro { get; set; }
    public int IdProcesso { get; set; }
    public string OrdemEncerrada { get; set; }
    public string DataInicio { get; set; }
    public string DataEntrega { get; set; }
    public string DataNegociada { get; set; }
    public bool UtilizaCracha { get; set; }
    public string PedidoVenda { get; set; }
    public string Servico { get; set; }
    public string DestinacaoODF { get; set; }
    public string NumeroPedido { get; set; }
    public string Programador { get; set; }
    public string Planejador { get; set; }
    public string Observacao { get; set; }
    public bool Retrabalho { get; set; }
    public int OdfDestino { get; set; }
    public int OdfFaturamento { get; set; }
    public string CodigoProdutoFaturamento { get; set; }
    public string Lote { get; set; }
    public bool OrdemFechada { get; set; }
    public string CodigoCliente { get; set; }

}

public class ProdutoDto
{
    public string Codigo { get; set; }
}

File: korp.qualidade.inspecaosaida/models/OrdemProducao.go
package models

import "github.com/shopspring/decimal"

type OrdemProducao struct {
	RecnoPpedlise          int
	ODF                    int
	OdfApontada            int
	CodigoProduto          string
	DescricaoProduto       string
	Situacao               string
	Revisao                string
	QuantidadeOrdem        decimal.Decimal
	QuantidadeProduzida    decimal.Decimal
	QuantidadeInspecionada decimal.Decimal
	QuantidadeLote         decimal.Decimal
	DataInicio             string
	DataEntrega            string
	DataEmissao            string
	Lote                   string
	NumeroPedido           string
	Cliente                string
	Plano                  string
	DataNegociada          string
	DescricaoPlano         string
	RecnoProcesso          int
}
